test/test.go

```diff
@@ -160,8 +160,9 @@ func (h *Helper) DoRun(args []string) error {
 	} else {
 		prog = filepath.Join(h.wd, "testdep"+ExeSuffix)
 	}
-	args = append(args[:1], append([]string{"-v"}, args[1:]...)...)
-	cmd := exec.Command(prog, args...)
+	newargs := []string{args[0], "-v"}
+	newargs = append(newargs, args[1:]...)
```

Comment:
The latter.  It was a modified slice that was getting recorded, messing up future tests.